compiler/one-cmds/one-quantize

```diff
@@ -308,10 +316,10 @@ def _quantize(args):
         # input and output path
         if _utils._is_valid_attr(args, 'input_path'):
             circle_quantizer_cmd.append(getattr(args, 'input_path'))
-        tmp_output_path_1 = os.path.join(
+        tmp_weights_fake_quant_path = os.path.join(
             tmpdir,
-            os.path.splitext(os.path.basename(args.input_path))[0]) + '1.circle'
```

Comment:
I renamed `~~1.circle` to `~~.weights_fake_quant.circle` for better readability.